Sources/Operation.swift

```diff
@@ -36,11 +49,17 @@ struct OperationBuilder: Builder {
     let responses: [Int : Reference<ResponseBuilder>]
     let defaultResponse: Reference<ResponseBuilder>?
     let deprecated: Bool
+    let operationId: String?
```

Comment:
Can do. Just though it might be nice to match the name of the property in swagger, that way consumers who know Swagger are can easily find them and also use autocomplete. What do you think? 
I agree that `id` or `identifier` is a better _Swift_ name